.github/workflows/phylogenetic.yaml

```diff
@@ -38,7 +40,56 @@ on:
         type: string
 
 jobs:
+  check-new-data:
+    if: |
+      github.event_name == 'workflow_run' &&
+      github.event.workflow_run.conclusion == 'success'
+    runs-on: ubuntu-latest
+    outputs:
+      cache-hit: ${{ steps.check-cache.outputs.cache-hit }}
+    steps:
+      - name: Get sha256sum
+        id: get-sha256sum
+        env:
+          AWS_DEFAULT_REGION: ${{ vars.AWS_DEFAULT_REGION }}
+        run: |
+          s3_urls=(
+            "s3://nextstrain-data/files/workflows/oropouche/metadata.tsv.zst"
+            "s3://nextstrain-data/files/workflows/oropouche/L/sequences.fasta.zst"
+            "s3://nextstrain-data/files/workflows/oropouche/M/sequences.fasta.zst"
+            "s3://nextstrain-data/files/workflows/oropouche/S/sequences.fasta.zst"
+          )
+
+          # Code below is modified from ingest/upload-to-s3
+          # https://github.com/nextstrain/ingest/blob/c0b4c6bb5e6ccbba86374d2c09b42077768aac23/upload-to-s3#L23-L29
+
+          no_hash=0000000000000000000000000000000000000000000000000000000000000000
+
+          for s3_url in "${s3_urls[@]}"; do
+            s3path="${s3_url#s3://}"
+            bucket="${s3path%%/*}"
+            key="${s3path#*/}"
+
+            s3_hash="$(aws s3api head-object --no-sign-request --bucket "$bucket" --key "$key" --query Metadata.sha256sum --output text 2>/dev/null || echo "$no_hash")"
+            echo "${s3_hash}" | tee -a ingest-output-sha256sum
+          done
+
+      - name: Check cache
+        id: check-cache
+        uses: actions/cache@v4
+        with:
+          path: ingest-output-sha256sum
+          key: ingest-output-sha256sum-${{ hashFiles('ingest-output-sha256sum') }}
+          lookup-only: true
+
   set_config_overrides:
+    needs: [check-new-data]
```

Comment:
Similar situation to https://github.com/nextstrain/nipah/pull/11#discussion_r2078101254

> The dependency chain of `check-new-data` -> `set_config_overrides` -> `phylogenetic` will prevent `phyologenetic` from running as expected (encountered this in [zika](https://github.com/nextstrain/zika/pull/81#issuecomment-2795072655)). See fix in https://github.com/nextstrain/zika/pull/83.

Reply:
...and force-pushed an update here too.